### PR TITLE
SMOKE on ARM

### PR DESCRIPTION
**Pull request by Yikun**

### CODE REVIEW

File: smoke.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+## Environment Requirements
+### Software Requirements
+| Item  | Version  |  Download Address |
+| ------------ | ------------ | ------------ |
+|  HDF5 |  1.10.1 | https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.1/src/hdf5-1.10.1.tar.gz  |
+|  PNETCDF |  1.8.0 |  https://parallel-netcdf.github.io/Release/parallel-netcdf-1.8.0.tar.gz |
+|  NETCDF-C | 4.4..1.1  |  https://github.com/Unidata/netcdf-c/archive/refs/tags/v4.4.1.1.tar.gz |
+| NETCDF-F  | 4.4.1  | https://github.com/Unidata/netcdf-fortran/archive/refs/tags/v4.4.1.tar.gz  |
+|  IOAPI | 3.2  | https://github.com/cjcoats/ioapi-3.2  |
+|  SMOKE | smoke_v481.Linux2_x86_64ifort.tar.gz smoke_v481.nctox.data.tar.gz smoke_install_v481.csh  | https://github.com/CEMPD/SMOKE/releases  |
```
Comment (martin-g) [inline smoke.md]: The content in the second column is not really the version.

File: smoke.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    yum install zlib* -y
+### Installing GNU 9.3
+
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "source /opt/rh/devtoolset-9/enable" >> /etc/profile
```
Comment (martin-g) [inline smoke.md:47]: `source` 👍🏼

File: smoke.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+### Installing Open MPI
```
Comment (martin-g) [inline smoke.md:48]: Add Open MPI to the table with dependencies above

**Review by martin-g — COMMENTED** [review]
Same improvements could be done as in the other PRs

File: smoke.md (hunk 1 above)
Comment (qinluyao761) [reply smoke.md]: done

File: smoke.md (hunk 3 above)
Comment (qinluyao761) [reply smoke.md:48]: done

**Pull request merged**